### PR TITLE
Add blogpost 2020-10/00

### PR DESCRIPTION
**Pull request by danielfoehrKn**
**What this PR does / why we need it**:

Add blogpost 2020-10/00 about the Shoot reconciliation flow.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

What is the archive date for? I added one for the end of the year but I am not sure.

**Release note**:

```improvement operator

```

### CODE REVIEW

File: website/blog/2020-10/00/_index.md
Comment (g-pavlov) [inline website/blog/2020-10/00/_index.md:102]: Would it make sense to maintain it in the gardener repo and benefit from mutual effort to keep it up-to-date?

File: website/blog/2020-10/00/_index.md
Comment (g-pavlov) [inline website/blog/2020-10/00/_index.md:107]: It seems worthy to place an image already here and let people take a look immediately. You mention that there are other formats so they can jump to that later if necessary. but probably 90% will be more comfortable to open a png right here.

```suggestion
You can find the diagram [on Github.com](https://github.com/danielfoehrKn/diagrams/tree/master/gardener/shoot-reconciliation). There are multiple formats available.

![gardener reconcile diagram](https://raw.githubusercontent.com/danielfoehrKn/diagrams/master/gardener/shoot-reconciliation/gardener_reconcile_with_grid.png)
```

**Review by g-pavlov — APPROVED** [review]
All good. Please, find two suggestions below

File: website/blog/2020-10/00/_index.md
Comment (g-pavlov) [reply website/blog/2020-10/00/_index.md:107]: or rather change the markdown tag  to  and keep the image small with style (400 width may be a good anchor). it needs to be a thumbnail hinting to open up for details not a regular picture.

File: website/blog/2020-10/00/_index.md
Comment (danielfoehrKn) [reply website/blog/2020-10/00/_index.md:102]: My hope had been that it could become such a thing or referenced in the documentation even. 
I do not think that this will be the case in short term. 
I already asked if people want to contribute/check it, but people are mostly busy. 

Maybe if more people find this useful (externals too) we can strive to add it as an additional repository. 
But for now I would not want to create a repo in the official organization that will probably eventually be abandoned.

File: website/blog/2020-10/00/_index.md
Comment (danielfoehrKn) [reply website/blog/2020-10/00/_index.md:107]: I tried that, but I cannot find a way to insert the diagram there that is readable, providing value, at all. Probably too large for any mobile or laptop device to "just view". 
Plus the picture will be outdated with the next commit to the repository. 

So not sure.

Comment (danielfoehrKn) [reply website/blog/2020-10/00/_index.md:107]: I am very much open to it if you got a setting that I could try out.

File: website/blog/2020-10/00/_index.md
Comment (g-pavlov) [reply website/blog/2020-10/00/_index.md:102]: There are other examples with a single maintainer too. I don't think that per-se is an issue and still leaves the door open for other contributors. But I leave this to you to decide.

File: website/blog/2020-10/00/_index.md
Comment (g-pavlov) [reply website/blog/2020-10/00/_index.md:107]: No the point is not to make it readable on the page, but visually hint that there's a (large) diagram if you click this thumbnail image, but you can do that here and not need to jump on another location. The point is, 100% of the people will look into the PNG first when they land on the other location anyway and quite likely it will be ok for most of them (you can zoom in/out and scroll the browser window). So UX advises that we spare them this additional click and look up for a format.

Maybe try this:
Some text</a>

Play with the width and remove the margin style (for centering) if necessary. If you are still convinced it's not ok, leave it be. That's just an improvement suggestion.

File: website/blog/2020-10/00/_index.md
Comment (danielfoehrKn) [reply website/blog/2020-10/00/_index.md:102]: Talked to the colleagues and we think it makes most sense to add it into the documentation repository itself. Will adjust the PR.

Comment (danielfoehrKn) [reply website/blog/2020-10/00/_index.md:102]: Agreed with @g-pavlov to leave it in the external repo.

File: website/blog/2020-10/00/_index.md
Comment (danielfoehrKn) [reply website/blog/2020-10/00/_index.md]: thanks added it!

**Comment by danielfoehrKn** [comment]
@g-pavlov  I am ready from my side. We might want to coordinate with the kubevirt blogpost that was opened today.

**Pull request merged**
